cmd/crank/beta/validate/image.go

```diff
@@ -72,6 +72,12 @@ type ErrBaseLayerNotFound struct {
 	error
 }
 
+// Is implements the errors.Is interface.
+func (e *ErrBaseLayerNotFound) Is(target error) bool {
```

Comment:
can you help me understand why we need to implement this function? why exactly doesn't the standard `errors.Is()` work already for us? 🤔 

when quickly searching, i wasn't finding too many examples of other folks needing to do this. thanks for the explanation!

Reply:
Yeah, not the most idiomatic code. I should be using errors.As directly from the IsErrBaseLayerNotFound.